pkg/descheduler/strategies/duplicates.go

```diff
@@ -85,9 +85,6 @@ func FindDuplicatePods(pods []*v1.Pod) DuplicatePodsMap {
 		if err != nil || sr == nil {
 			continue
 		}
```

Comment:
remove this check also.

Reply:
I need sr in the next line, so I had to include that.

Reply:
but the check is not needed i think, because all pods with sr==nil or err != nil are already checked in ListEvictablePodsOnNode so no pod at this stage should have sr==nil or err != nil.

Reply:
The reason is - I have to ignore err which is second argument and take sr only in order to remove this check. Will make the change now.